app/views/result_filters/subject/_form.html.erb

```diff
@@ -7,17 +7,14 @@
     <%= form_with(url: subject_path, method: :post, data: { 'ga-event-form' => 'Subject' }) do |f| %>
       <%= render 'shared/hidden_fields', exclude_keys: %w[subjects senCourses], form: f %>
       <h1 class="govuk-heading-l" data-qa="heading"><%= I18n.t('page_titles.subjects_filter') %></h1>
-      <div class="accordion govuk-form-group" data-module="govuk-accordion">
+
+      <%= govuk_accordion(id: 'subject-area-accordion') do |accordion| %>
         <% @subject_areas.each_with_index do |subject_area, counter| %>
-          <div class="govuk-accordion__section<%= if subject_area_is_selected?(subject_area: subject_area) || (counter.zero? && no_subject_selected_error?) then ' govuk-accordion__section--expanded' end %>" data-qa="subject_area">
-            <div class="govuk-accordion__section-header">
-              <h2 class="govuk-accordion__section-heading" data-qa="subject_area__name">
-                <button type="button" data-qa="subject_area__accordion_button" id="accordion-heading-<%= subject_area.typename %>" aria-controls="<%= subject_area.typename.downcase %>-content-<%= counter %>" class="govuk-accordion__section-button" aria-expanded="<%= subject_area_is_selected?(subject_area: subject_area) || (counter.zero? && no_subject_selected_error?) ? 'true' : 'false' %>">
-                  <%= subject_area.name %>
-                </button>
-              </h2>
-              <span class="govuk-accordion__icon" aria-hidden="true"></span>
-            </div>
+          <%= accordion.add_section(
+            title: subject_area.name,
+            expanded: subject_area_is_selected?(subject_area: subject_area) || (counter.zero? && no_subject_selected_error?),
+            html_attributes: { 'data-qa': 'subject_area' },
+          ) do %>
             <div id="<%= subject_area.typename.downcase %>-content-<%= counter %>" class="govuk-accordion__section-content" aria-labelledby="accordion-heading-<%= subject_area.typename.downcase %>">
```

Comment:
Is `govuk-accordion__section-content` not part of the component already, or are overriding this somehow? Where does the `add_section` method come from? (I had a look at the component code and couldn’t quite work it out).

Reply:
Actually, inspecting the code, and seeing how it renders, we actually have `govuk-accordion__section-content` nested within `govuk-accordion__section-content` which doubles the space between the section header and content:

| Production | This PR |
| - | - |
| <img width="660" alt="Screenshot 2021-03-07 at 14 37 27" src="https://user-images.githubusercontent.com/813383/110243578-e753b280-7f52-11eb-80e1-f5942c6757a9.png"> | <img width="660" alt="Screenshot 2021-03-07 at 14 37 13" src="https://user-images.githubusercontent.com/813383/110243573-e6228580-7f52-11eb-9e69-111d39f67a66.png"> |

Here’s a screenshot of the offending code:

<img width="579" alt="Screenshot 2021-03-07 at 14 37 44" src="https://user-images.githubusercontent.com/813383/110243579-e753b280-7f52-11eb-8e68-21ecdc9f06c6.png">

My side senses were right, `govuk-accordion__section-content` shouldn’t feature in any code outside of the component.

Reply:
Good spot, I think that's fixed now